src/main/java/peoplesoft/logic/commands/job/JobAssignCommand.java

```diff
@@ -1,102 +1,128 @@
 package peoplesoft.logic.commands.job;
 
 import static java.util.Objects.requireNonNull;
-import static peoplesoft.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static peoplesoft.commons.util.CollectionUtil.requireAllNonNull;
 import static peoplesoft.logic.parser.CliSyntax.PREFIX_INDEX;
 
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 
 import peoplesoft.commons.core.Messages;
 import peoplesoft.commons.core.index.Index;
 import peoplesoft.logic.commands.Command;
 import peoplesoft.logic.commands.CommandResult;
 import peoplesoft.logic.commands.exceptions.CommandException;
-import peoplesoft.logic.parser.ArgumentMultimap;
-import peoplesoft.logic.parser.ParserUtil;
-import peoplesoft.logic.parser.exceptions.ParseException;
-import peoplesoft.logic.parser.job.JobAssignCommandParser;
 import peoplesoft.model.Model;
 import peoplesoft.model.employment.Employment;
 import peoplesoft.model.employment.exceptions.DuplicateEmploymentException;
 import peoplesoft.model.job.Job;
 import peoplesoft.model.person.Person;
-import peoplesoft.model.util.ID;
 
 /**
  * Assigns a {@code Job} to a {@code Person}.
  */
 public class JobAssignCommand extends Command {
 
-    //TODO: change if needed
     public static final String COMMAND_WORD = "assign";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Assigns a job to a person. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Assigns a job to one or more person(s). "
             + "Parameters: "
-            + "JOBID "
-            + PREFIX_INDEX + "INDEX ";
+            + "JOB_INDEX "
+            + PREFIX_INDEX + "PERSON_INDEX [PERSON_INDEX]...";
 
-    public static final String MESSAGE_SUCCESS = "Assigned Job %s to %s\n%s has the following jobs: %s";
-    public static final String MESSAGE_JOB_NOT_FOUND = "This job does not exist";
-    public static final String MESSAGE_DUPLICATE_EMPLOYMENT = "This person is already assigned to this job";
+    public static final String MESSAGE_SUCCESS = "Assigned Job %s to %s.";
+    public static final String MESSAGE_DUPLICATE_EMPLOYMENT =
+            "A total of %s persons have existing employments.";
```

Comment:
"%s people have been assigned to jobs" may sound better

Reply:
Changed to "%s person(s) have already been assigned to this job."